Stripe/STPAddressFieldTableViewCell.m

```diff
@@ -64,6 +64,14 @@ - (instancetype)initWithType:(STPAddressFieldType)type
         NSString *countryCode = [[NSLocale autoupdatingCurrentLocale] objectForKey:NSLocaleCountryCode];
         NSMutableArray *otherCountryCodes = [[NSLocale ISOCountryCodes] mutableCopy];
         [otherCountryCodes removeObject:countryCode];
+        [otherCountryCodes sortUsingComparator:^NSComparisonResult(NSString *code1, NSString *code2) {
+            NSString *localeID1 = [NSLocale localeIdentifierFromComponents:@{NSLocaleCountryCode: code1}];
+            NSString *localeID2 = [NSLocale localeIdentifierFromComponents:@{NSLocaleCountryCode: code2}];
+            NSLocale *locale = [NSLocale autoupdatingCurrentLocale];
```

Comment:
Probably should pull out the locale outside the block for the weird race condition of it changing mid-sort?